stop_and_dwell_strategic_plugin/test/test_stop_and_dwell_strategic_plugin.cpp

```diff
@@ -36,12 +36,12 @@ class DummyNode :  public carma_ros2_utils::CarmaLifecycleNode
     carma_ros2_utils::CallbackReturn handle_on_configure(const rclcpp_lifecycle::State &){return CallbackReturn::SUCCESS;}
 };
 
-
+/**
```

Comment:
Can we add the comment here for this commented out test?

Reply:
added